services/account/apps/api/src/controllers/v1/handles-v1.controller.ts

```diff
@@ -69,6 +75,32 @@ export class HandlesControllerV1 {
     }
   }
 
+  @Get('change/:newHandle')
+  @HttpCode(HttpStatus.OK)
+  @ApiOperation({ summary: 'Get a properly encoded ClaimHandlePayload that can be signed.' })
+  @ApiOkResponse({ description: 'ClaimHandlePayload request enqueued' })
+  /**
+   * Using the provided query parameters, creates a new payload that can be signed to change handle.
+   * @param queryParams - The query parameters for changing the handle.
+   * @returns Payload is included for convenience. Encoded payload to be used when signing the transaction.
+   * @throws An error if the change handle payload creation fails.
+   */
+  async getChangeHandlePayload(@Param('newHandle') newHandle: string): Promise<ChangeHandlePayloadRequest> {
+    try {
+      const expiration = await this.handlesService.getExpiration();
+      const payload = { baseHandle: newHandle, expiration };
+      const encodedPayload = await this.handlesService.encodePayload(payload);
```

Comment:
So, this gives us a SCALE-encoded payload. But we need to sign raw bytes. So what we actually need to return is a Byte-wrapped array of bytes, encoded as a hex string. Like so:
```suggestion
      const encodedPayload = u8aToHex(u8aWrapBytes(this.handlesService.encodePayload(payload).toU8a()));
```

Also, `encodePayload` shouldn't be `async`, so we don't need `await`.